notebooks/feature_engineering/labs/4_keras_adv_feat_eng.ipynb

```diff
@@ -31,15 +31,23 @@
    "metadata": {},
```

Comment:
<p>Add back a paragraph describing the exercise in a markdown cell.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>